src/packages/auth/open-id-connect-auth/use-oidc.tsx

```diff
@@ -5,40 +5,46 @@ import { storeToken } from './token-utils';
 import { useEffect, useState } from 'react';
 import { UsersApi } from '../../sdk/users-api';
 
-type LoginOidcComponentTypes = {
+type OidcWrapperTypes = {
 	WrappedComponent: any;
 	saveUserProps: ({ roles, stamp }: { roles: string[]; stamp: string }) => void;
 };
 
-const LoginOidcComponent = ({
+export const LoginWrapper = ({
 	WrappedComponent,
-	saveUserProps,
-}: LoginOidcComponentTypes) => {
-	const { isUserLoggedIn, login, oidcTokens } = useOidc({
+}: Omit<OidcWrapperTypes, 'saveUserProps'>) => {
+	const { isUserLoggedIn, login } = useOidc({
 		assertUserLoggedIn: false,
 	});
-	const { renewTokens } = useOidc({ assertUserLoggedIn: true });
-	const [userInformationLoaded, setUserInformationLoaded] = useState(false);
 
 	if (!isUserLoggedIn) {
 		login({
 			doesCurrentHrefRequiresAuth: true,
 		});
-		return null;
 	}
 
+	return <WrappedComponent />;
```

Comment:
Do really need to reutrn this component ? Why not only a return null ? Because the component won't be displayed ?